pkg/utils/utils.go

```diff
@@ -150,3 +151,12 @@ func ParsePKey(pKey string) (int, error) {
 
 	return int(i), nil
 }
+
+// ParseNetworkId returns the network name and network namespace
+func ParseNetworkId(networkId string) (string, string, error) {
+	separatorIndex := strings.LastIndex(networkId, ".")
```

Comment:
why not use strings.Split?

Reply:
Network CRD name may contain "." but the k8s namespace is not allowed to contain ".", if there is a network with name `ib.sriov` in `default` namespace then networkId would be `ib.sriov.default`, that will cause the `ParseNetworkId` to fail

Reply:
namespace name needs to be a FQDN, you can use `_` to separate between the namespace and network then use split.
the *dot* can appear in network as well i think

creating a namespace with an `_` will fail so its safe to assume that

```
root@r-dcs31:~# kubectl apply -f /tmp/namespace.yaml
The Namespace "my_namespace" is invalid: metadata.name: Invalid value: "my_namespace": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```